proposals/NNNN-modify-the-range-of-APT's-maxDuration.md

```diff
@@ -0,0 +1,79 @@
+# Modify the range of APT's maxDuration

```

Comment:
Please update the title of this PR to match this proposal title, so `maxDuration` should be capitalized in the title of the PR.

Reply:
@theresalech Thanks, I have updated the title of this PR.